llvm/lib/Target/RISCV/AsmParser/RISCVAsmParser.cpp

```diff
@@ -943,7 +943,8 @@ struct RISCVOperand final : public MCParsedAsmOperand {
     RISCVMCExpr::VariantKind VK = RISCVMCExpr::VK_RISCV_None;
     int64_t Imm;
     bool IsConstantImm = evaluateConstantImm(getImm(), Imm, VK);
-    return IsConstantImm && isShiftedInt<7, 5>(Imm) &&
+    return IsConstantImm &&
```

Comment:
This function seems to be used in RISCVInstrInfoZicbo.td for a non branch/jump instruction. Do you maybe want to make this a separate change?

Reply:
Sorry, yeah, missed the prefetch testcases. Will add them imminently.

Reply:
Done